src/tasks/index.ts

```diff
@@ -0,0 +1,174 @@
+import type { Schema } from 'ajv';
+import * as cronParser from 'cron-parser';
+import { tasks as tasksEnv } from '../config-loader/env';
+import type * as Db from '../database-layer/db';
+import { BadRequestError } from '../sbvr-api/errors';
+import { addPureHook } from '../sbvr-api/hooks';
+import { PinejsClient } from '../sbvr-api/sbvr-utils';
+import type * as sbvrUtils from '../sbvr-api/sbvr-utils';
+import { ajv, apiRoot, channel } from './common';
+import type { TaskHandler } from './types';
+import { Worker } from './worker';
+
+export * from './types';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const modelText: string = require('./model.sbvr');
+
+export const config = {
+	models: [
+		{
+			modelName: apiRoot,
+			apiRoot,
+			modelText,
+			customServerCode: exports,
+		},
+	] as sbvrUtils.ExecutableModel[],
```

Comment:
`satisfies` is better because it doesn't cast in the same way
```suggestion
	] satisfies sbvrUtils.ExecutableModel[],
```
although I think it'd be better to use
```ts
export const config ConfigLoader.Config = {
```
so that the whole thing is typed and matches the expectation